### ISSUE TITLE
Не нужно ли добавить в словарик имён примеры склонения?

### ISSUE THREAD

**Issue by SelenIT** [web-standards-ru/dictionary#228]
Например, «Анне ван Кестерен» — «блог Анне ван Кестерена», «Джен Симмонс» — «статья Джен Симмонс». По крайней мере в случаях, когда ни по имени, ни по фамилии нельзя сходу понять, по какому правилу они склоняются.

**Comment by Sigiller** [comment]
> По крайней мере в случаях, когда ни по имени, ни по фамилии нельзя сходу понять, по какому правилу они склоняются (и склоняются ли).

В таких случаях определенно стоит.

**Comment by pepelsbey** [comment]
Да, стоит это сделать по примеру того, как в словаре терминов приводятся примеры употребления
